src/shared/parse.cpp

```diff
@@ -337,7 +328,7 @@ static void Parse_CreatePunctuationTable( script_t *script, punctuation_t *punct
 	if ( !script->punctuationtable )
 	{
 		script->punctuationtable = ( punctuation_t ** )
-		                           Z_Malloc( 256 * sizeof( punctuation_t * ) );
+		                           BG_Alloc( 256 * sizeof( punctuation_t * ) );
```

Comment:
The commit description could be more precise e.g. "Replace Z_Malloc with BG_Alloc in parse.cpp"